src/model/services/controlplane/appcatalogs/appcatalogs.ts

```diff
@@ -0,0 +1,57 @@
+import { HttpClient } from 'model/clients';
+import {
+  IAppCatalog,
+  IMetaData,
+  ISpec,
+  IStorage,
+} from 'model/services/controlplane/appcatalogs/types';
+import { getBaseConfiguration } from 'model/services/controlplane/base';
+
+/**
+ * Get the all the app catalogs in the installation.
+ * @param client - The HTTP client.
+ */
+export async function getAppCatalogs(
+  client: HttpClient
+): Promise<IAppCatalog[]> {
+  const { ApplicationGiantswarmIoV1alpha1Api } = await import(
```

Comment:
Importing this dynamically, to save up a couple hundred of KBs from the final bundle size (Which also means we won't have dead Control Plane API code when the feature is not supported).